xml/System.Net.Security/SslClientAuthenticationOptions.xml

```diff
@@ -179,9 +199,15 @@
         <ReturnType>System.Security.Cryptography.X509Certificates.X509CertificateCollection</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Collection of certificates to be considered for client's authentication to server.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
```

Comment:
I'm not seeing this section in the preview for some reason: https://review.docs.microsoft.com/en-us/dotnet/api/system.net.security.sslclientauthenticationoptions.clientcertificates?view=netcore-3.1&branch=pr-en-us-4366

Reply:
I added missing <remarks> tag. Preview is correct now.